packages/components/src/services/hds-theming.ts

```diff
@@ -0,0 +1,233 @@
+import Service from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export enum HdsThemeValues {
+  // system settings (prefers-color-scheme)
+  System = 'system',
+  // user settings for dark/light
+  Light = 'light',
+  Dark = 'dark',
+}
+
+enum HdsModesBaseValues {
+  Hds = 'hds', // TODO understand if it should be `default`
+}
+
+enum HdsModesLightValues {
+  CdsG0 = 'cds-g0',
+  CdsG10 = 'cds-g10',
+}
+
+enum HdsModesDarkValues {
+  CdsG90 = 'cds-g90',
+  CdsG100 = 'cds-g100',
```

Comment:
I still don't really understand the purpose of having multiple Light & Dark value options.

Reply:
@KristinLBradley I just noticed that I didn't explain this in the sync. Essentially, since Carbon supports 4 themes/modes, we should do the same (in case some consumer, for reasons we didn't anticipate, may need to use a different combination that the standard `g0/g100`).

Reply:
Ok, I understand we need to or should follow how it's done in Carbon, but I find it confusing.